vendor/github.com/urfave/cli/docs.go

```diff
@@ -1,3 +1,5 @@
+//+build !cli_minimal
```

Comment:
I would abandon the `cli_minimal` tag and just use `//+build ignore`. This way `make` and `go build` are consistent.

Reply:
Agree. Especially `go test -run Something` is really annoying if the tags don't work.

Reply:
This is a great suggestion :+1: